restful_api/controllers/2

```diff
@@ -0,0 +1,32 @@
+#-*- coding: utf-8 -*-
+from odoo import http
+from odoo.http import request
+import json
+
+class RestfulApi(http.Controller):
+    @http.route('/restful_api/restful_api', auth='public')
+    def index(self, **kw):
+        vendor = request.env['res.partner'].search_count([])
+        return "Totally %s vendor avilable now!"
+
+    @http.route('/api/products', type='json', csrf=False, auth='public', methods=['GET', 'POST'])
+    def product_list(self, **kw):
+        data = [{
+                'id': drug.id,
```

Comment:
Wasn't this API supposed to be only for CUSTOMER / PATIENT Data. But it seems to also have "drug" details in it? Is this PR taking care of more data elements that just customer sync?

Reply:
it is being retained for further development,now the code has been removed.